### PR TITLE
Decoders version bump

### PR DESCRIPTION
**Pull request by lu-zero**
- [x] needs a valid aom 2.0.0 package, so far I couldn't find a .deb that works with ubuntu.

### CODE REVIEW

**Comment by lu-zero** [comment]
This should be squashed on merge.

**Comment by Luni-4** [comment]
We need to update `aom` for `travis` too.

**Comment by lu-zero** [comment]
@Luni-4 will you take care of it?

File: .travis/install-aom.sh [hunk 1]
```diff
@@ -1,27 +1,21 @@
 #!/bin/bash
 set -ex
 
-AOM_VERSION="1.0.0.errata1-3~18.04.york0"
-PKG_URL="http://ppa.launchpad.net/jonathonf/ffmpeg-4/ubuntu/pool/main/a/aom"
-
-case "$ARCH" in
-  x86_64) ARCH=amd64 ;;
```
Comment (vibhoothiiaanand) [inline .travis/install-aom.sh:8]: Why are we removing x86_64 case, as it is in GitHub Actions?

Comment (Luni-4) [reply .travis/install-aom.sh:8]: Exactly, we are testing it with `GitHub Actions`

**Pull request merged**